modules/graphdb/templates/04_gdb_conf_overrides.sh.tpl

```diff
@@ -37,12 +37,20 @@ log_with_timestamp "Writing configuration files"
 # graphdb.external-url.enforce.transactions: determines whether it is necessary to rewrite the Location header when no proxy is configured.
 # This is required because when working with the GDB transaction endpoint it returns an erroneous URL with HTTP protocol instead of HTTPS
 if [ "${node_count}" -eq 1 ]; then
+
+  if [ "${disable_agw}" = "true" ]; then
+  EXTERNAL_URL="https://${graphdb_external_address_fqdn}${context_path}/"
```

Comment:
Instead of having `${graphdb_external_address_fqdn}${context_path}/` which heavily depends on the var.context_path why not accept any value for var.context_path and trim it here to what you need for example
"https://${graphdb_external_address_fqdn}/${context_path}/"

example:
```
CLEAN_CONTEXT_PATH=$(echo "${context_path}" | sed 's#^/*##' | sed 's#/*$##')

if [ "${disable_agw}" = "true" ]; then
  EXTERNAL_URL="https://${graphdb_external_address_fqdn}/${CLEAN_CONTEXT_PATH}/"
else
  EXTERNAL_URL="https://${graphdb_external_address_fqdn}/"
fi
```

this way you can pass anything to context_path and will end up as expected regardless of /

Reply:
My idea was to have the variable as close as possible to the option in the Application Gateway. The example there is "/api, /test/" or something. 
This way the variable would be similar to the application gateway example - context path = "/example". 
I could change it with the regex as you suggested if you think it would be easier for the users.

Reply:
Well with the regex whatever they put in will be transformed to the thing we need, less possibility for a user mistake, and we can get rid of the validation in the variable.